bin/datatool/src/util.rs

```diff
@@ -564,3 +561,140 @@ fn get_genesis_block_info(genesis_json: &str) -> anyhow::Result<BlockInfo> {
         stateroot: genesis_stateroot,
     })
 }
+
+/// Retrieves the genesis L1 view either from a Bitcoin RPC client or from a local file.
+///
+/// When the `btc-client` feature is enabled, this function connects to a Bitcoin node
+/// using the RPC credentials from `cmd` and fetches the genesis L1 view at the specified
+/// block height (defaults to 100 if not provided).
+///
+/// When the `btc-client` feature is disabled, the function loads the genesis L1 view
+/// from a JSON file specified in `cmd.genesis_l1_view_file`.
+///
+/// # Arguments
+/// * `cmd` - Command parameters containing Bitcoin RPC connection details and file paths
+///
+/// # Returns
+/// * `Ok(GenesisL1View)` - The successfully retrieved genesis L1 view
+/// * `Err(anyhow::Error)` - If RPC connection fails, file reading fails, or JSON parsing fails
+fn get_genesis_l1_view(cmd: &SubcParams) -> anyhow::Result<GenesisL1View> {
+    #[cfg(feature = "btc-client")]
+    {
+        // When the btc-client feature is enabled, we can fetch the genesis L1 view from a Bitcoin
+        // node.
```

Comment:
It should only look to these options if the view file is not set.  If the view file is set and any of these options are, then it should use that and print a warning.